pages/badges/badge-single.jsx

```diff
@@ -236,6 +229,40 @@ var BadgePage = React.createClass({
     );
   },
 
+  showBackToBadgeModal: function() {
+    var canClose = this.state.canCloseModal;
+    var error = this.state.applicationError;
+    var disabled = "disabled";
+
+    if (canClose || error) {
+      disabled = null;
+    }
+
+    if (error) {
+      return (
+        <Modal className="modal-credly error folded" hideModal={this.state.canCloseModal? this.hideApplyModal : false}>
```

Comment:
sorry, I meant in the ternary condition, can we add a space between the condition and the `?` operator?